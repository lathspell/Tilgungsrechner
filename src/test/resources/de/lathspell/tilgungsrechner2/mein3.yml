months:
    "2016-04":
        darlehensbetrag: 1000
        sollzins: 0
        monatsrate: 100
        auszahlung: 1000
    