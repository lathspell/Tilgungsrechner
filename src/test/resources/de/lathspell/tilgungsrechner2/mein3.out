# mein3
Tilgungsplan
Periode 	Rate (EUR) 	Zinsanteil (EUR) 	Tilgungsanteil (EUR) 	Restschuld am Periodenende (EUR)
Apr 2016           100,00               0,00                 100,00                    900,00
Mai 2016           100,00               0,00                 100,00                    800,00
Jun 2016           100,00               0,00                 100,00                    700,00
Jul 2016           100,00               0,00                 100,00                    600,00
Aug 2016           100,00               0,00                 100,00                    500,00
Sep 2016           100,00               0,00                 100,00                    400,00
Okt 2016           100,00               0,00                 100,00                    300,00
Nov 2016           100,00               0,00                 100,00                    200,00
Dez 2016           100,00               0,00                 100,00                    100,00
Jan 2017           100,00               0,00                 100,00                      0,00
GESAMT: 10 Monatsraten: 1.000,00   davon Zinsen: 0,00  für ein Darlehen über 1.000,00
